Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -295,6 +295,27 @@ Advanced Configuration
 This section covers a few advanced configuration modes for the kube-proxy replacement
 that go beyond the above Quick-Start guide and are entirely optional.
 
+Policy Enforcement
+******************
+
+When accessing a service from outside the cluster, the :ref:`arch_id_security`
+assignment depends on the routing and the kube-proxy replacement configuration.
+
+In the tunneling mode (i.e., ``--tunnel=vxlan`` or ``--tunnel=geneve``), the request
+to the service will have the ``reserved:world`` security identity. This allows a user to
+use a network policy with the ``fromEntities: cluster`` label selector.
+
+In the direct-routing mode (i.e., ``--tunnel=disabled``), the security identity
+will be set to ``reserved:world`` if the request was sent to the node which runs the
+selected endpoint by the LB. If not, i.e., the request needs to be forwarded to
+another node after the service endpoint selection, then it will have either ``reserved:host``
```

Comment:
I'm not sure that this will ever show up as `reserved:host`, have you seen this? Even with `--enable-remote-node-identity=false`, I'm pretty sure that Cilium would consider the traffic as `remote-node` identity. IIRC the difference is that if `--enable-remote-node-identity=false` then Cilium will automatically allow the traffic (via "allow from host" logic)

Reply:
Oh, my previous reading of `--enable-remote-node-identity=false` was that any node2node traffic will be identified as `reserved:host` regardless of whether the sender node is the same as the destination node.